Content.Client/Tips/ClippyUIController.cs

```diff
@@ -0,0 +1,281 @@
+using Content.Client.Gameplay;
+using System.Numerics;
+using Content.Client.Message;
+using Content.Client.Paper;
+using Content.Shared.CCVar;
+using Content.Shared.Movement.Components;
+using Content.Shared.Tips;
+using Robust.Client.GameObjects;
+using Robust.Client.ResourceManagement;
+using Robust.Client.State;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controllers;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.Audio;
+using Robust.Shared.Configuration;
+using Robust.Shared.Console;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+using static Content.Client.Tips.ClippyUI;
+
+namespace Content.Client.Tips;
+
+public sealed class ClippyUIController : UIController
+{
+    [Dependency] private readonly IStateManager _state = default!;
+    [Dependency] private readonly IConsoleHost _conHost = default!;
+    [Dependency] private readonly IPrototypeManager _protoMan = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] private readonly IResourceCache _resCache = default!;
+    [UISystemDependency] private readonly AudioSystem _audio = default!;
+    [UISystemDependency] private readonly EntityManager _entSys = default!;
+
+    public const float Padding = 50;
+    public static Angle WaddleRotation = Angle.FromDegrees(10);
+
+    private EntityUid _entity;
+    private float _secondsUntilNextState;
+    private int _previousStep = 0;
+    private ClippyEvent? _currentMessage;
+    private readonly Queue<ClippyEvent> _queuedMessages = new();
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        _conHost.RegisterCommand("local_clippy", ClippyCommand);
```

Comment:
Unsure on this one, but having the command in a different file using `LocalizedCommands` is more clear.